examples/protonect/src/opengl_depth_packet_processor.cpp

```diff
@@ -347,10 +424,10 @@ struct OpenGLDepthPacketProcessorImpl : public WithOpenGLBindings
   Texture<U16C1> input_data;
 
   Texture<F32C4> stage1_debug;
-  Texture<F32C3> stage1_data[3];
+  Texture<F32C4> stage1_data[3];
```

Comment:
The effect is the texture can't be attached to FBO. With `INTEL_DEBUG=fbo,...` you get `FBO incomplete: Unsupported HW texture/renderbuffer format attached: MESA_FORMAT_RGB_FLOAT32`